README.md

```diff
@@ -60,36 +60,35 @@ The following dependecies can be installed for you by the extension by clicking
 ## Useful Links
 
 - Tutorials and Example Code for Adafruit CPX:
-  - Adafruit CPX library tutorial: (https://learn.adafruit.com/circuitpython-made-easy-on-circuit-playground-express/circuit-playground-express-library)
-  - Adafruit CPX Examples on GitHub: (https://github.com/adafruit/Adafruit_CircuitPython_CircuitPlayground/tree/master/examples)
-  - Adafruit CPX Guided Tour (Intro for the Hardware) (https://learn.adafruit.com/adafruit-circuit-playground-express/guided-tour)
+  - [Adafruit CPX library tutorial](https://learn.adafruit.com/circuitpython-made-easy-on-circuit-playground-express/circuit-playground-express-library)
+  - [Adafruit CPX Examples on GitHub](https://github.com/adafruit/Adafruit_CircuitPython_CircuitPlayground/tree/master/examples)
+  - [Adafruit CPX Guided Tour (Intro for the Hardware)](https://learn.adafruit.com/adafruit-circuit-playground-express/guided-tour)
 - Format Adafruit CPX device:
-  - Tutorial for formatting Adafruit CPX for CircuitPython (https://learn.adafruit.com/welcome-to-circuitpython/installing-circuitpython)
-  - Download Firmware .uf2 file (https://learn.adafruit.com/adafruit-circuit-playground-express/circuitpython-quickstart)
-  - Download the latest version of the Adafruit CPX library (link: https://learn.adafruit.com/welcome-to-circuitpython/circuitpython-libraries)
+  - [Tutorial for formatting Adafruit CPX for CircuitPython](https://learn.adafruit.com/welcome-to-circuitpython/installing-circuitpython)
+  - [Download Firmware .uf2 file](https://learn.adafruit.com/adafruit-circuit-playground-express/circuitpython-quickstart)
+  - [Download the latest version of the Adafruit CPX library](https://learn.adafruit.com/welcome-to-circuitpython/circuitpython-libraries)
 
 ## How to use
 
 To use Device Simulator Express, install the extension from the marketplace and reload VS Code.
 
 ### 1. Start with the “New File” Command.
 
-1. Type in Device Simulator Express: New File” in the command palette(`CTRL+SHIFT+P`to open the command palette).  
-   !["New File" animation](https://www.microsoft.com/en-us/garage/wp-content/uploads/sites/5/2019/08/newFile.gif)
-2. Name and save your file somewhere, and we’re good to go!(3)
-3. Start with some examples: you can find examples files and tutorials inside the comments,
-   as well as in the notification pop up when you run the `“Device Simulator Express: New File”` Command.
+1. Type in Device Simulator Express: New File” in the command palette (`CTRL+SHIFT+P` to open the command palette).  

```

Comment:
Are quotation marks missing before "Device Simulator Express: New File"?

Reply:
Good catch, fixed in next commit